webapp/channels/src/actions/emoji_actions_load_recent.test.ts

```diff
@@ -75,12 +76,14 @@ describe('loadRecentlyUsedCustomEmojis', () => {
 
         await store.dispatch(Actions.loadRecentlyUsedCustomEmojis());
 
-        expect(store.getActions()).toEqual([
-            {
-                type: EmojiTypes.RECEIVED_CUSTOM_EMOJIS,
-                data: [emoji1, emoji2],
-            },
-        ]);
+        await waitFor(() => {
```

Comment:
Anywhere that we're testing logic that relies on `getCustomEmojisByNameBatched` now, this needs to be wrapped in a `waitFor`.

Interestingly, `waitFor` still works for non-React code even though it comes from RTL